openeo/Sentinel-3.ipynb

```diff
@@ -0,0 +1,444 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "21ec5555",
+   "metadata": {},
+   "source": [
+    "# Sentinel-3\n",
+    "\n",
+    "\n",
+    "This notebook explores working with Sentinel-3 data. \n",
+    "\n",
+    "## Sentinel-3 OLCI\n",
+    "\n",
+    "https://sentinels.copernicus.eu/web/sentinel/user-guides/sentinel-3-olci\n",
+    "\n",
+    "The OLCI dataset provided by Sentinelhub is based on the level-1b products. These products are provided in \"instrument\" projection rather than being projected into a ground-based reference system. Hence, these products do not come with a 'native' reference system. The openEO layer is currently configured to use EPSG:4326 unprojected coordinates, with a resolution set to a fixed value that tries to approximate the native 300m ground resolution.\n"
```

Comment:
> The openEO layer

In openEO we avoid the concept "layer", because it could man both "collection" or "band"

Reply:
updated as collections